litex/soc/interconnect/avalon.py

```diff
@@ -9,6 +9,156 @@
 from migen import *
 
 from litex.soc.interconnect import stream
+from . import wishbone
```

Comment:
Please use similar imports than other LiteX files (`from litex.soc.interconnect import wishbone`). Relative imports could be fine but I we switch to it, I have a preference to do it for the whole project. We could think about it in the future.

Reply:
done